src/redux/notes-slice.ts

```diff
@@ -1,35 +1,59 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { Comment, Note, NotesState } from 'redux/notes-slice.types';
+import { v4 } from 'uuid';
+
+const initialState: NotesState = {
+  notes: [],
+};
 
 const notesSlice = createSlice({
   name: 'notesPage',
-  initialState: {
-    notes: [
-      {
-        id: 1,
-        name: 'shopping list',
-        content: 'need buy milk',
-        comments: [{ id: 1, author: 'John White', content: 'and buy meet', created_at: '21.12.2022' }],
-      },
-      {
-        id: 2,
-        name: 'tasks for the day',
-        content: 'meet with Katy in 21:00',
-        comments: [
-          { id: 1, author: 'Katy Ling', content: 'come in 20:30', created_at: '24.12.2022' },
-          { id: 2, author: 'Katy Ling', content: 'sorry, come in 20:00', created_at: '24.12.2022' },
-        ],
-      },
-    ],
-  },
+  initialState,
   reducers: {
     addNote(state, action) {
-      state.notes.push(action.payload);
+      const newNote: Note = {
+        id: v4(),
```

Comment:
Does not FB generates id for documents, I think it should ?
Please try to use id from FB

Reply:
not connected FB